pym/portage/dep/__init__.py

```diff
@@ -562,6 +562,9 @@ def missing_white_space_check(token, pos):
 					basestring):
 					if stack[level][-1] == "||" and not l:
 						#Optimize: || ( ) -> .
+						if not eapi_attrs.empty_groups_always_true:
+							# in EAPI 7+, we need to fail here
+							l.append(token_class("__const__/empty-any-of"))
```

Comment:
We should update the token_default to  `token_class=_unicode`, for calls that do not specify token_class like in pym/portage/package/ebuild/_config/LicenseManager.py. Ultimately we'll need some special user interface handling, but this is fine for now I guess.

Reply:
And done!